api.rst

```diff
@@ -1839,6 +1841,42 @@ You can also call a function that takes a single parameter of type JSON by sendi
 
   8
 
+.. _s_proc_single_unnamed:
+
+Calling functions with a single unnamed parameter
+-------------------------------------------------
+
+You can make a POST request to a function with a single unnamed parameter to send raw ``json/jsonb``, ``bytea`` or ``text`` data.
+
+To send raw ``json/jsonb``, you can avoid using the ``Prefer: params=single-object`` header if the function has a single unnamed JSON parameter and the header ``Content-Type: application/json`` is included in the request.
```

Comment:
Should we deprecate `params=single-object` or is there still a valid use-case for it?

Also, I think the fallback behaviour for the overloaded json case should be mentioned, WDYT?

Reply:
> Should we deprecate params=single-object or is there still a valid use-case for it?

The unnamed json parameter doesn't work for GET, while `params=single-object` does.

Also, right now unnamed json works for webhooks that only send a JSON body. There are other webhooks that also send some query params besides the JSON body and this makes the function call fail(maybe the solution is to add a second unnamed json param for "absorbing" the querystring parameters).

In any case, I think it's too early to deprecate(next version could), we need the new unnamed json feature to be proven first.

Reply:
Makes a ton of sense!